tools/java/org/unicode/cldr/test/SubmissionLocales.java

```diff
@@ -42,7 +42,9 @@
 //                }
 //            }
 
-    public static final Pattern ALLOWED_IN_LIMITED_PATHS = null; 
+    public static final Pattern ALLOWED_IN_LIMITED_PATHS = 
+    Pattern.compile("//ldml/annotations/annotation.*+[🤵👰⬆➡⬇⬅♾✖➕➖➗]");
```

Comment:
This does not seem to work as intended (in smoketest). Should this line instead be
` Pattern.compile("//ldml/annotations/annotation\\[@cp=\"([🤵👰⬆➡⬇⬅♾✖➕➖➗])\"");`